docs/reference/uds-core/idam/customization.md

```diff
@@ -71,6 +71,42 @@ kubectl create configmap keycloak-theme-overrides \
   --from-file=favicon.png=path/to/local/directory/favicon.png
 ```
 
+### Branding customizations
+
+In a similar theme to Branding customizations, the UDS Identity Config supports adjusting the Terms and Conditions (if enabled).
+
+These customizations require overriding the Keycloak Helm Chart provided by the UDS Core. Here's an example:
+
+```yaml
+packages:
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: x.x.x
+    overrides:
+      keycloak:
+        keycloak:
+          values:
+            - path: themeCustomizations
+              value:
+                resources:
+                  termsAndConditions:
+                    text:
+                       configmap:
+                          key: text
+                          name: keycloak-theme-overrides
+```
+
+The configuration a single text override inside the customized theme. The example above uses a single key `text` which is expected to exist in the `keycloak-theme-overrides` ConfigMap. The value of this key is base64 encoded text that corresponds to the Terms and Conditions. The text follows the Keycloak Theme override format used in FTL files (for more information, see the [Keycloak documentation](https://www.keycloak.org/docs/latest/server_development/index.html)) and can contain HTML formatting. You can create it using the following command:
```

Comment:
Maybe should link freemarker manual here? https://freemarker.apache.org/docs/index.html